carLot.js

```diff
@@ -0,0 +1,54 @@
+var CarLot = (function () {
+  var inventory = [];
+
+  return {
+
+    getInventory: function () {
+      return inventory;
+    },
+
+    loadInventory: function (cb) {
+      var inventoryLoader = new XMLHttpRequest();
+
+      inventoryLoader.open('GET', 'inventory.json');
+      inventoryLoader.addEventListener("load", function (evt) {
+        inventory = JSON.parse(evt.target.responseText).cars;
+        cb(inventory);
+      });
+      inventoryLoader.send();
+    },
+
+    activateEvents: function() {
+      var carBlock = document.querySelectorAll('.car-block');
+      var input = document.querySelector('input');
+      var descript;
+
+      carBlock.forEach(e => e.addEventListener('click', changeBlock));
+      input.addEventListener('keypress', updateDescription);
+
+      function changeBlock(evt){
```

Comment:
This function should be augmented in it's own IIFE as per the instructions. It should also take the DOM element that was clicked on and the color.